cmake/PythonNumpyPybind11.cmake

```diff
@@ -12,15 +16,16 @@ include_directories(${NUMPY_INCLUDE_DIR})
 
 include(FetchContent)
 
-message("Using pybind11 version: ${PYBIND11_VERSION}")
```

Comment:
Why removing this message?

Reply:
by accident 😅  Thanks!

Reply:
Wait. Did I remove it? I think I just changed the case?